### ISSUE TITLE
Feature request: provide the final branch name as an action output

### ISSUE THREAD

**Issue by mshick** [robvanderleek/create-issue-branch#201]
I was looking to chain some functionality off of this in GitHub Actions, notably adding a link to open a PR using the branch name CIB creates. While I can recreate the routine for generating similar branch names, it would be ideal to get the generated branch name as an output, via `setOutput`.

**Comment by robvanderleek** [comment]
Hi @mshick 

That sounds interesting, I had not considered the option before to chain CIB with other GitHub Actions.
I've added an output variable for the branch name, please see the [Readme](https://github.com/robvanderleek/create-issue-branch#github-action-output-variable).

Hope this works for your use-case, if not let me know. Am also interested to hear about your Actions workflow when it's complete.

**Comment by mshick** [comment]
Awesome! Thanks. 

Fantastic app/action; really appreciate it.

The use case I have is forming up a follow-on comment with a link to create a PR that includes a PR template as a query param. It's the best solution I've found so far for making use of PR templates. I lifted your routines here as a temporary workaround: https://github.com/takeshape/github-action-create-pr-comment/. Now that I have your branchName output I can probably replace this action with some simple `run` scripting.

I'll be sure to share when I have things updated.
